README.md

```diff
@@ -39,6 +40,94 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
+## Development 
+
+#### 1. Install [Go](https://golang.org/doc/install) and [dep](https://github.com/golang/dep), if you haven't already.
+#### 2. Clone this repository. 
+#### 3. Setup configuration files: 
+* `politeiad` and `politeiawww` both have configuration files that you should set up to make execution easier. 
+* You should create the configuration files under the following path:  
+  - **Mac OS** `/Users/<username>/Library/Application Support/Politeiad/politeiad.conf`
+  - **Windows** `C:\Users\<username>\AppData\Local\Politeiad/politeiad.conf`
+  - **Ubuntu** `~/.politeiad/politeiad.conf`
+  
+* Copy and change the  [`sample-politeiawww.conf`](https://github.com/decred/politeia/blob/master/politeiawww/sample-politeiawww.conf) and [`sample-politeiad.conf`](https://github.com/decred/politeia/blob/master/politeiad/sample-politeiad.conf) files.
+* You may use the following default configurations:
+
+**politeiad.conf**: 
+`rpcuser=user rpcpass=pass testnet=true`	
+
+
+**politeiawww.conf**:
+
+    rpchost=127.0.0.1
+    rpcuser=user
+    rpcpass=pass
+    rpccert="/Users/<username>/Library/Application 
+    Support/Politeiad/https.cert"
+    proxy=true
+    testnet=true
+
+**Attention:** The rpccert path is referenced on MacOs path. See step 4 for more OS paths.
+#### 4.Start the Politeiad server by running on your terminal:
+
+    politeiad
+
+#### 5. Download server identity to client:
+
+    politeia -v -testnet -rpchost 127.0.0.1 identity
+
+Accept default path by pressing `enter`.
+
+Result should look something like this:
+
+```
+FQDN       : localhost
+Nick       : politeiad
+Key        : dfd6caacf0bbe5725efc67e703e912c37931b4edbf17122947a1e0fcd9755f6d
+Identity   : 99e748e13d7ecf70ef6b5afa376d692cd7cb4dbb3d26fa83f417d29e44c6bb6c
+Fingerprint: medI4T1+z3Dva1r6N21pLNfLTbs9JvqD9BfSnkTGu2w=
+
+Save to /Users/<username>/Library/Application Support/Politeia/identity.json or ctrl-c to abort
+Identity saved to: /Users/<username>/Library/Application Support/Politeia/identity.json
+```
+
+#### 6. Start the Politeiawww server by running on your terminal:
+
+    politeiawww
+
+**Awesome! From this point you have your politeia server up running!**
+#### 7. Running the politeiawww reference client:
+* With politeiad and politeiawww running type on your terminal:
```

Comment:
Could you add a note in here that says something like: "Make sure you do not have an email server set up for `politeiawww`, because `politeiawww_refclient` will not execute correctly. So before you execute `politeiawww_refclient`, make sure to comment out or remove the following config options and restart `politeiawww`: `mailhost`, `mailuser`, `mailpass`, `webserveraddress`."